components/stratum/mining.c

```diff
@@ -162,22 +154,17 @@ double test_nonce_value(const bm_job *job, const uint32_t nonce, const uint32_t
     return ds;
 }
 
-uint32_t increment_bitmask(const uint32_t value, const uint32_t mask)
+uint32_t increment_bitmask(uint32_t value, uint32_t mask)
```

Comment:
why did you modify increment_bitmask that is only used by BM1397 (which does not do hardware version rolling) ?

Reply:
Given that the BM1366 ASIC chips do use hardware version rolling, and this functionality is critical for the target setup, i should leave increment_bitmask as it was, without altering its behavior.
Thus, no change is required for the increment_bitmask function, and we should maintain its original implementation, assuming it is only used by non-version-rolling chips (BM1397).